spec/models/organization_spec.rb

```diff
@@ -8,15 +8,44 @@
   it { should validate_uniqueness_of(:login) }
 
   describe 'load_organizations' do
+    let(:whitelisted_organisations) { ['test-organization'] }
+    before do
+      stub_const("Organization::WHITELISTED_ORGS", whitelisted_organisations)
+    end
+
     context 'creates new organization' do
-      let(:whitelisted_organisations) { ['test-organization'] }
+      it 'create a new Organization record' do
+        expect { Organization.load_organizations }.to change{ Organization.count }.by(1)
+      end
+    end
 
+    context 'updates organization' do
+      it 'updates fields if they were changed' do
+        Organization.load_organizations
+        organization = Organization.find_by_login('test-organization')
+        organization.name = 'old_name'
+        organization.github_url = 'https://github.com/old_organization_url'
+        organization.save!
```

Comment:
This should clearly be replaced with a factory. Like this:

```ruby
create(:organization, name: 'old_name', …)
Organization.load_organizations
expect(Organization.count).to eq(1)
organization = Organization.first
expect(organization.name).to eq('new_name')
```
and so on.

You should avoid duplicating your application code in tests, this is extremely error-prone